lib/oracledb.js

```diff
@@ -675,7 +678,11 @@ function getPool(poolAlias) {
 function initOracleClient(arg1) {
   let options = {};
   errors.assertArgCount(arguments, 0, 1);
+  let binaryPath = undefined
   if (arg1 !== undefined) {
+    if ( arg1.binaryPath !== undefined) {
+      binaryPath = arg1.binaryPath
```

Comment:
Why create a separate variable? Why not simply pass the options to `initCLib()`?